bin/cron/upload_to_firebase

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env ruby
+
+# This script fetches weather forecast data from api.openweathermap and uploads to the shared firebase channel.
+require_relative '../../deployment'
+require 'cdo/only_one'
+require 'net/http'
+require 'json'
+require_relative '../../shared/middleware/helpers/firebase_helper'
+
+WeatherForecastOffice = Struct.new(:city, :state, :zip)
+WeatherRecord = Struct.new(:id, :city, :state, :timestamp, :temp_min, :temp_max, :weather, :icon, :wind_speed)
+
+def get_weather_data
+  forecast_offices = [
+    WeatherForecastOffice.new("Birmingham", "Alabama", "35040"),
+    WeatherForecastOffice.new("Eureka", "California", "95501"),
+    WeatherForecastOffice.new("Miami", "Florida", "33165"),
+    WeatherForecastOffice.new("St. Louis", "Missouri", "63304"),
+    WeatherForecastOffice.new("Wakefield", "Virginia", "23888")
+  ]
+  records = []
+  id = 1
+  forecast_offices.each do |office|
+    puts office
+    url = "http://api.openweathermap.org/data/2.5/forecast?zip=#{office.zip},us&units=imperial&appid=c719e74bc77b787d940386405e0cbb83"
+    response = Net::HTTP.get_response(URI(url))
+    next unless response.code == '200'
+    measurements = JSON.parse(response.body)["list"]
+    measurements.each do |m|
+      row = WeatherRecord.new(
+        id,
+        office.city,
+        office.state,
+        m["dt_txt"],
+        m["main"]["temp_min"],
+        m["main"]["temp_max"],
+        m["weather"][0]["description"],
+        m["weather"][0]["icon"],
+        m["wind"]["speed"]
+      )
+      records.push(row.to_h.to_json)
+      id += 1
+    end
+  end
+  return records
+end
+
+def main
+  fb = FirebaseHelper.new('shared')
+  weather_records = get_weather_data
+  fb.delete_shared_table('openweathermap')
+  fb.upload_shared_table('openweathermap', weather_records)
```

Comment:
Do we ever expect these data sets to get large enough where we would update rather than delete and replace?

Reply:
We want to fully replace the data, so we don't want `update`. Looks like we probably don't need `delete` though, because `set` fully replaces the data itself.